libs/core/type_support/include/hpx/type_support/pack.hpp

```diff
@@ -177,22 +173,23 @@ namespace hpx { namespace util {
         {
         };
 
+        template <std::size_t J>
+        static empty at_index_check(...);
```

Comment:
@K-ballo this is what you meant?

Reply:
Yes, now this overload will be included in the `at_index_check<I>` call.